grammar/TypeQL.g4

```diff
@@ -332,35 +332,46 @@ DATETIME_       : DATE_FRAGMENT_ 'T' TIME_              ;
 
 VAR_CONCEPT_            : VAR_CONCEPT_ANONYMOUS_ | VAR_CONCEPT_NAMED_   ;
 VAR_CONCEPT_ANONYMOUS_  : '$_' ;
-VAR_CONCEPT_NAMED_      : '$'  TYPE_CHAR_H_ TYPE_CHAR_T_* ;
-VAR_VALUE_              : '?'  TYPE_CHAR_H_ TYPE_CHAR_T_* ;
+VAR_CONCEPT_NAMED_      : '$'  IDENTIFIER_VAR_H IDENTIFIER_VAR_T* ;
+VAR_VALUE_              : '?'  IDENTIFIER_VAR_H IDENTIFIER_VAR_T* ;
 IID_                    : '0x' [0-9a-f]+                  ;
-LABEL_                  : TYPE_CHAR_H_ TYPE_CHAR_T_*      ;
+LABEL_                  : IDENTIFIER_LABEL_H IDENTIFIER_LABEL_T*      ;
```

Comment:
`TYPE_` prefix was a hangover from the past when "label" was called "type" in the grammar. Fixed here.